### DISCUSSION TITLE
Demo does not work. Say wrong user or password

### DISCUSSION THREAD

**Discussion by Havouza15** [mealie-recipes/mealie#3051] [General]
Hi!
I want to test the demo before I make an install on my server. But login is impossible, say user password is invalid

**Comment by michael-genson** [comment]
Hey there,

Can you share the URL you're using for the demo, as well as the username and password that you're trying? I just checked and I was able to get in with no problem.

The demo is hosted at https://demo.mealie.io/ and the username and password are posted on there:
```
Username: changeme@example.com
Password: MyPassword
```

**Reply by Havouza15** [reply]
OK it was the demo I tested but it did not work before. I just copied the logi. But works now
